services/IBF-pipeline/pipeline/lib/pipeline/dynamicDataDb.py

```diff
@@ -91,31 +91,13 @@ def processDynamicDataDb(self):
     def apiGetRequest(self, path, country_code):
         import requests
 
-        login_response = requests.post('http://12.0.0.8:3000/api/user/login', data=[('email',ADMIN_LOGIN),('password',ADMIN_PASSWORD)])
+        login_response = requests.post('http://ibf-api-service:3000/api/user/login', data=[('email',ADMIN_LOGIN),('password',ADMIN_PASSWORD)])
         TOKEN = login_response.json()['user']['token']
 
-        response = requests.get('http://12.0.0.8:3000/api/'+path+'/'+country_code,headers={'Authorization': 'Bearer ' + TOKEN})
+        response = requests.get('http://ibf-api-service:3000/api/'+path+'/'+country_code,headers={'Authorization': 'Bearer ' + TOKEN})
```

Comment:
The api url string should be configurable. Preferably as an ENV variable.

Reply:
@gulfaraz I get that it's good to make it a configurable setting, but it's not server-dependent in this case right? Can it not go in settings.py?